src/encoded/schemas/library.json

```diff
@@ -101,83 +101,36 @@
                 "type": "string"
             }
         },
-        "fragment_maximum_length": {
-            "title": "Fragment Maximum Length",
-            "description": "Maximum length of the sequenced fragments, e.g. as predicted by Agilent Bioanalyzer (bp)",
-            "type": "integer",
-            "minimum": 0
-        },
-        "fragment_mean_length": {
-            "title": "Fragment Mean Length",
-            "description": "Mean length of the sequenced fragments, e.g. as predicted by Agilent Bioanalyzer (bp)",
-            "type": "number",
-            "minimum": 0
-        },
-        "fragment_minimum_length": {
-            "title": "Fragment Minimum Length",
-            "description": "Minimum length of the sequenced fragments, e.g. as predicted by Agilent Bioanalyzer (bp)",
-            "type": "integer",
-            "minimum": 0
-        },
-        "fragment_standard_deviation_length": {
-            "title": "Fragment Standard Deviation Length",
-            "description": "Standard deviation of length of the sequenced fragments, e.g. as predicted by Agilent Bioanalyzer (bp)",
-            "type": "number",
-            "minimum": 0
-        },
-        "insert_coefficient_of_variation": {
-            "title": "Insert Coefficient of Variation",
-            "description": "Coefficient of variation of the length of the sample molecule in the fragments to be sequenced",
-            "type": "number",
-            "minimum": 0
-        },
-        "insert_maximum_length": {
-            "title": "Insert Maximum Length",
-            "description": "Maximum length of the sample molecule in the fragments to be sequenced (bp)",
-            "type": "integer",
-            "minimum": 0
-        },
-        "insert_mean_length": {
-            "title": "Insert Mean Length",
-            "description": "Mean length of the sample molecule in the fragments to be sequenced (bp)",
-            "type": "number",
-            "minimum": 0
-        },
-        "insert_minimum_length": {
-            "title": "Insert Minimum Length",
-            "description": "Minimum length of the sample molecule in the fragments to be sequenced (bp)",
-            "type": "integer",
-            "minimum": 0
```

Comment:
So we don't want QC data on the library fragment sizes anymore? We do have items with these values in prod, and that data won't be easily retrievable once these changes go in.

Reply:
Do you think it would make sense to add this data in comments for the data that has information for these properties?

Reply:
Not really, as I'm not a huge fan of collecting very unstructured data since hard to parse/read and not searchable. If we're removing the properties because we don't think the information is relevant, then the values can be deleted; otherwise, would keep some subset of the properties.

Reply:
I think I was a little overzealous with removing all of these. New commit keeps all but four of these properties.